### PR TITLE
Temp feature a

### PR DESCRIPTION
**Pull request by tomaszszturmajconsileon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-        <title></title>
+        <title>Hello World!</title>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link href="css/style.css" rel="stylesheet">
     </head>
     <body>
         <h1>Hello world!</h1>
+
+
```
Comment (tszturmaj) [inline index.html]: Please remove unnecessary line.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+        <p>Welcome on Hello World site!</p>
+
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (tszturmaj) [inline index.html]: Could you do this w/o jQuery, I think this is simple think and we don't want unnecessary external dependecies.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+$(document).ready(function () {
```
Comment (tszturmaj) [inline main.js]: > Could you do this w/o jQuery, I think this is simple think and we don't want unnecessary external dependecies.

File: index.html (hunk 1 above)
Comment (tomaszszturmajconsileon) [inline index.html]: Ohh sorry..

File: index.html (hunk 2 above)
Comment (tomaszszturmajconsileon) [inline index.html]: Yes you have right, it is unnecessary.

**Pull request merged**